dbt_superset_lineage/push_descriptions.py

```diff
@@ -166,17 +168,32 @@ def merge_columns_info(dataset, tables):
             'id': sst_column['id']
         }
 
-        # add column descriptions
+        # add column descriptions and labels
         if column_name in dbt_columns \
                 and 'description' in dbt_columns[column_name] \
                 and (sst_column['expression'] is None  # database columns
                      or sst_column['expression'] == ''):
+            
             description = dbt_columns[column_name]['description']
             description = convert_markdown_to_plain_text(description)
         else:
             description = sst_column['description']
+        
         column_new['description'] = description
 
+        # Check if 'meta' field exists in dbt_columns
```

Comment:
Maybe you could merge this with a previous `if` to keep the code DRY? It basically does the same operations.

Reply:
Would you mind reflecting this one, please?

Reply:
is this better?

Reply:
Why do you check for expressions when getting column descriptions